src/meta/types/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(provide_any)]
```

Comment:
This FEATURE behind the license is wried :)

Reply:
+1